packages/core/custom-elements-manifest.config.mjs

```diff
@@ -0,0 +1,8 @@
+import { readonlyPlugin } from 'cem-plugin-readonly';
+import { moduleFileExtensionsPlugin } from 'cem-plugin-module-file-extensions';
+
+export default {
+  globs: ['./**/*.d.ts'],
+  litelement: true,
+  plugins: [readonlyPlugin(), moduleFileExtensionsPlugin({ from: /\.d\.(t|j)sx?$/, to: '.js' })],
+};
```

Comment:
The tool that generates the custom elements metadata has a config option. This enables plugins like the readonly which helps mark TS specific syntax like readonly props in the metadata.